test/hotspot/jtreg/compiler/types/TestSubTypeCheckMacroTrichotomy.java

```diff
@@ -24,10 +25,13 @@
 /**
  * @test
  * @bug 8253566
+ * @bug 8295414
  * @summary clazz.isAssignableFrom will return false for interface implementors
  * @requires vm.compiler2.enabled
  *
  * @run main/othervm -XX:-BackgroundCompilation TestSubTypeCheckMacroTrichotomy
+ * @run main/othervm -XX:-BackgroundCompilation -XX:+StressReflectiveCode -XX:+ExpandSubTypeCheckAtParseTime
```

Comment:
You should additionally add `-XX:+UnlockDiagnosticVMOptions` since `ExpandSubTypeCheckAtParseTime` is a diagnostic VM flag.

Reply:
Thanks for catching that.  Unfortunately, it also requires a debug build, but if I add @requires vm.debug, then the tests won't run with release builds.  I went with the next best option: -XX:+IgnoreUnrecognizedVMOptions.

Reply:
Yes, you're right. I agree that this is the best solution in that case.